### ISSUE TITLE
[Feature Request][Desktop][Android] Ahead-of-time compilation support

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#1291]
## Feature Description
Please consider Ahead-of-time compilation support for Desktop and Android platforms like it was done for iOS.
It increase performance and AIR applications will be more robust and "feels" more like "native", especially for low-performance devices.
Also it could decrease AIR applications file size.

More discussions about that:
https://github.com/airsdk/Adobe-Runtime-Support/discussions/1269#discussioncomment-1488451
https://github.com/airsdk/Adobe-Runtime-Support/discussions/1269#discussioncomment-1488460

## Known Workarounds
none

**Comment by itlancer** [comment]
@ajwfrost 
Any progress on this?

**Comment by ajwfrost** [comment]
Yes, we'd been working on this, and the bulk of the code is there in terms of the runtime. Just need to work out how to invoke it - a very different start-up process from the normal bootstrapping - and then get the SDK tools updated for it...

Will try to get someone to look into these final aspects..

**Comment by itlancer** [comment]
@ajwfrost 
Will it be available in AIR 51.2 or 52?

**Comment by itlancer** [comment]
@ajwfrost, any news about that?

**Comment by bobaoapae** [comment]
@ajwfrost ANy news on AOT?

**Comment by ajwfrost** [comment]
For the Android AOT work - we have everything building for this now, although the creation of the packages is still a little manual rather than being something that ADT already handles. But there's a runtime issue where the AOT-compiled definitions that are linked in are not being found by the runtime properly, so we need to do some debugging and comparison between the start-up on iOS vs Android (which has the added complication of being split between the two languages, Java and C++).

So, it's progressing from time to time, but needs a bit more of a push to get this to a releasable state.

thanks
